docs/install-run/special/docker.txt

```diff
@@ -4,138 +4,45 @@
 Run CrateDB on Docker
 =====================
 
-Docker allows developers to package applications and dependencies into
-standard, shippable units, great for easy deployments across platforms and
-constantly changing needs. CrateDB is available as a Docker image and is a
-perfect data store for Docker based applications.
+CrateDB and Docker_ are a great match thanks to CrateDB's shared-nothing,
+horizontally scalable architecture that lends itself well to containerization.
 
-if you don't have Docker setup yet, find `instructions for all platforms here
-<https://docs.docker.com/engine/installation/>`_.
+.. _Docker: https://www.docker.com/
 
 .. NOTE::
```

Comment:
A great thing about using Docker is that you don't need to care about the environment. Therefore I think it's not necessary to mention the requirement of Java 8 here.
However, we could mention that the `crate` Docker image is based upon Alpine Linux which minimizes the container size.

Reply:
Maybe even link the `Dockerfile` in [crate/docker-crate](https://github.com/crate/docker-crate/blob/master/Dockerfile)?

Reply:
that's not strictly true. because the JVM version is still important, for setting resource limits and so on

also, because Java is proprietary software, I think it's important that we mention it

we mention the requirement for Java 8 in all of our quick start guides, so there's an argument for consistency also (though, this is a weak argument)

Reply:
there is actually a difference: when you install crate from package or tarball you have java as requirement, but if you want to run it with docker, then the only dependency on your host is the docker environment.

Reply:
@chaudum pushed a fixup. could you re-review please? ty!